src/Stratis.Features.FederatedPeg/CollateralChecker.cs

```diff
@@ -77,6 +77,7 @@ public async Task InitializeAsync()
             foreach (CollateralFederationMember federationMember in this.federationManager.GetFederationMembers()
                 .Cast<CollateralFederationMember>().Where(x => x.CollateralAmount != null && x.CollateralAmount > 0))
             {
+                this.logger.LogTrace("Initializing federation member {0} with amount {1}.", federationMember.CollateralMainchainAddress, federationMember.CollateralAmount);
```

Comment:
All non-entry and non-exit logs should bee on debug level

Reply:
Will Change